eb_sqs/worker/service.py

```diff
@@ -44,9 +44,6 @@ def process_queues(self, queue_names):
         static_queues = queues
         last_update_time = timezone.now() - timedelta(seconds=settings.REFRESH_PREFIX_QUEUES_S)
 
-        self.write_healthcheck_file()
```

Comment:
would suggest to leave it here, or maybe even move it to the beginning of the function